README.md

```diff
@@ -5,6 +5,31 @@ A curated list of awesome [remote working](http://en.wikipedia.org/wiki/Telecomm
 
 *To contribute, click README.md and then the pencil icon. Make your changes and click the "Propose file change" button to submit a pull request. Make sure to follow the contributions guidelines in CONTRIBUTING.md.*
 
+## Table of Contents
+
+<!-- MarkdownTOC depth=4 -->

```

Comment:
It is not, in case links are kept updated manually. 
Those are just artefacts of https://packagecontrol.io/packages/MarkdownTOC